tests/acceptance/30_custom_promise_types/05_meta_attr.cf

```diff
@@ -0,0 +1,99 @@
+######################################################
+#
+#  Basic test of custom promise types / promise modules checking that using a
+#  'meta' attribute in combination with a custom promise works (i.e. doesn't
+#  crash).
+#
+#####################################################
+body common control
+{
+    inputs => { "../default.cf.sub" };
+    bundlesequence  => { default("$(this.promise_filename)") };
+    version => "1.0";
+}
+
+#######################################################
+
+bundle agent init
+{
+  files:
+    "$(G.testfile)"
+      delete => init_delete;
+}
+
+body delete init_delete
+{
+      dirlinks => "delete";
+      rmdirs   => "true";
+}
```

Comment:
```suggestion
bundle agent init
{
  files:
    "$(G.testfile)"
      delete => tidy;
}
```

Reply:
I like the test to be usable even without some body definitions from somewhere else.